extension/view/js/translation-notification-fxtranslations.js

```diff
@@ -200,57 +148,13 @@ window.MozTranslationNotification = class extends MozElements.Notification {
     this.state = this.translationNotificationManager.TranslationInfoBarStates.STATE_TRANSLATING;
   }
 
-  _close() {
-
-    this.close();
-
-    /*
-     * todo: this.translation is undefined
-     * const from = this._getSourceLang();
-     * const to = this._getTargetLang();
-     * this.translation.infobarClosed(from, to);
-     */
-  }
-
   /*
    * to be called when the infobar should be closed per user's wish (e.g.
    * by clicking the notification's close button, the not now button or choosing never to translate)
    */
   closeCommand() {
     this.translationNotificationManager.reportInfobarEvent("closed");
-    this._close();
-  }
-
-  /*
-   * to be called when the infobar should be closed per user's wish
-   * by clicking the Not now button
-   */
-  notNow() {
-    this.translationNotificationManager.reportInfobarEvent("not_now");
```

Comment:
same question about these two

Reply:
Same situation as above: neither `showOriginal()` neither `notNow()` are being called because although this time they are properly attached to the events in the forms, the panel they are part of won't ever be displayed because these functionalities were removed. 

It's not blatantly clear in the code in this file, since the state is controlled by the translationNotificationManager, but by using the extension is easy to notice that these controls are not ever being dsiplayed (we don't need notNow() since there are already the button to dismiss the infobar at the right end", and showOriginal() is not supported by the parser and actually it doesn't make much sense for me and we probably won't implement.